### ISSUE TITLE
Token field is missing from AddProjectHookOptions

### ISSUE THREAD

**Issue by sindrepm** [xanzy/go-gitlab#121]
The [Add project hook](https://docs.gitlab.com/ce/api/projects.html#add-project-hook) API endpoint supports a `token` parameter, however, this property is missing from the `AddProjectHookOptions` and `EditProjectHookOptions` types.

Can you please update these structs to include the Token field?

```go
type AddProjectHookOptions struct {
	URL                   *string `url:"url,omitempty" json:"url,omitempty"`
	PushEvents            *bool   `url:"push_events,omitempty" json:"push_events,omitempty"`
	IssuesEvents          *bool   `url:"issues_events,omitempty" json:"issues_events,omitempty"`
	MergeRequestsEvents   *bool   `url:"merge_requests_events,omitempty" json:"merge_requests_events,omitempty"`
	TagPushEvents         *bool   `url:"tag_push_events,omitempty" json:"tag_push_events,omitempty"`
	NoteEvents            *bool   `url:"note_events,omitempty" json:"note_events,omitempty"`
	BuildEvents           *bool   `url:"build_events,omitempty" json:"build_events,omitempty"`
	PipelineEvents        *bool   `url:"pipeline_events,omitempty" json:"pipeline_events,omitempty"`
	WikiPageEvents        *bool   `url:"wiki_page_events,omitempty" json:"wiki_page_events,omitempty"`
	EnableSSLVerification *bool   `url:"enable_ssl_verification,omitempty" json:"enable_ssl_verification,omitempty"`
	Token                 *string `url:"token,omitempty" json:"token,omitempty"` 
}
```

```go
type EditProjectHookOptions struct {
	URL                   *string `url:"url,omitempty" json:"url,omitempty"`
	PushEvents            *bool   `url:"push_events,omitempty" json:"push_events,omitempty"`
	IssuesEvents          *bool   `url:"issues_events,omitempty" json:"issues_events,omitempty"`
	MergeRequestsEvents   *bool   `url:"merge_requests_events,omitempty" json:"merge_requests_events,omitempty"`
	TagPushEvents         *bool   `url:"tag_push_events,omitempty" json:"tag_push_events,omitempty"`
	NoteEvents            *bool   `url:"note_events,omitempty" json:"note_events,omitempty"`
	BuildEvents           *bool   `url:"build_events,omitempty" json:"build_events,omitempty"`
	PipelineEvents        *bool   `url:"pipeline_events,omitempty" json:"pipeline_events,omitempty"`
	WikiPageEvents        *bool   `url:"wiki_page_events,omitempty" json:"wiki_page_events,omitempty"`
	EnableSSLVerification *bool   `url:"enable_ssl_verification,omitempty" json:"enable_ssl_verification,omitempty"`
	Token                 *string `url:"token,omitempty" json:"token,omitempty"` 
}
```

According to the API docs, the response does not include the token so I don't think there is a need to add this field to the `ProjectHook` type.

**Comment by svanharmelen** [comment]
If you can just paste those snippets into a PR, I'll be happy to merge it for you :wink: If you are unable to do so, then let me know and I'll will take a look later next week...

**Comment by sindrepm** [comment]
@svanharmelen I tried to create a branch and push, but I don't have access.

```
remote: Permission to xanzy/go-gitlab.git denied to sindrepm.
fatal: unable to access 'https://github.com/xanzy/go-gitlab.git/': The requested URL returned error: 403
```

**Comment by sindrepm** [comment]
Done: PR #122

**Comment by svanharmelen** [comment]
Fixed by PR #122
